---
layout: page
title: Opportunities
description:
background: '/img/Banner_LabLogo.jpg'
---

<body>

<h1>Research Opportunities</h1>
<hr style="border: 1px solid #f00"/>

	<h2>Phylogenetics</h2>
	<hr>
		<h3>Relationship of NGS read mapping success and phylo distance</h3>
		<hr style="border-top: dashed 1px;"/>
		For non-model organisms, projects that use next-generation sequencing (NGS) data are  challenging due to the lack of reference genomes available. This requires the mapping of sequence data to a species with a genome available or the sequencing and assembly of a reference-quality genome (the latter of which is sometimes financially and logistically unfeasible). Mapping is the step in bioinformatic data processing where NGS reads are aligned to a reference genome. It is understood that with increased evolutionary distance between the read and reference taxa the mapping success decreases, however the exact relationship of this decrease is not characterized. For this project, we map sequence data from four lizard species in the genus <em>Aspidoscelis</em> to reference genomes across the vertebrate tree of life and examine the relationship of mapping success to phylogenetic distance.

		<h3>Codivergence</h3>
		<hr style="border-top: dashed 1px;"/>

			<h4>Comparing accuracy of phylogenetic methods for estimating codivergence</h3>
				<p>Various natural forces can cause two independent lineages to diverge simultaneously (e.g., island fragmentation, aquatic isolation). Several phylogenetic software packages can estimate the probability of these occurances, and differences in their fundamental methodoligies may result in some being more accurate than others. We compared estimations between packages using simulated data, and are now examining these methods using empirical data (from "legacy" datasets).</p>

		<hr style="border-top: dashed 1px;"/>
		<h3>Mitonuclear Phylogenetics</h3>
		<hr style="border-top: dashed 1px;"/>
			<p>In Eukaryotic organisms, DNA resides inside both the nucleus and the mitochondria. Both of these genomes contain information that encodes cellular gene products (e.g., proteins and functional RNA). All of the gene products from the mitochondrial genome perform functions within the mitochondria, whereas the vast majority of gene products from the nuclear genome do not. However, a subset of gene products from the nuclear genome target the mitochondria; these gene products must interact with  those of the mitochondrial genome. Incompatibilities between the gene products can result in decreased mitochondrial function, which can negatively affect cellular, organal, and organismal performance</p>

		<hr style="border-top: dashed 1px;"/>
		<h3>Evolutionary Correlation</h3>
		<hr style="border-top: dashed 1px;"/>

	<h2>Genomics</h2>

		<h3>Differential Gene Expression</h3>
		<p>	Parthenogenetic lizards present a great model to understand the effects of selection and phenotypic plasticity in a non-recombining species with very little genetic variation. We sequenced transcriptomes from three populations of a parthenogenetic whiptail species (<em>Aspidoscelis tesselatus</em>) and will examine how gene expression in skeletal muscle tissue varies across the populations (which are located at different latidutidnal positions along the Rio Grande river basin in New Mexico, USA. Additionally, for one of the populations we collected samples from three different tissue types (skeletal muscle, liver, and heart) to examine differences in expression patterns across the tissues.</p>
		<h3>Mitonuclear Mismatching in Hybrid Parthenogens</h3>
		<p>		Some instances of natural hybridization in reptiles led to the formation of new species that reproduce asexually (Barley et al., 2022). In these species, offspring are produced without male contribution (i.e., these species are all female). For these asexual lizards, their genomes are locked in a hybrid state (Vrijenhoek & Pfeiler, 2008; Warren et al., 2018) (i.e., no sexual recombination occurs to alter their genetic composition). Asexual whiptail lizards exhibit reduced mitochondrial respiration (Cullum, 1997;Klabacka et al., 2022). Explicit examination of the molecular mechanisms involved in mito-nuclear mismatch is needed to understand which genes are responsible for reduced organism health in hybrid species.</p>
		<p>For this project we will bioinformatically examine the vast network of genes involved in mitochondrial respiration of hybrid asexual organisms (<em>Aspidoscelis tesselatus</em>) at three levels: (1) nucleotide sequence [DNA], (2) transcript sequence/abundance [RNA], and (3) amino acid sequence [protein]. Examination of these genes will help us understand (A) the specific genetic underpinnings of reduced mitochondrial respiration in hybrid asexual organisms, (B) whether hybrid asexual organisms can cope with mito-nuclear mismatch by “turning off” (silencing) or “turning down” (suppressing) genes that aren’t co-evolved with their mitochondria, and (C) the role of mito-nuclear interactions in speciation.</p>
		<h3>Population Genomics</h3>
		In biological study systems where multiple populations can be assigned to distinguishable categories (e.g., A and B), approaches to quantify genetic divergence between the categories should incorporate divergence between the populations themselves. To do this, calculations of divergence for pairwise population comparisons can be binned into “within” (comparisons of populations in the same category; e.g., A1–A2 and B1–B2) or “between” (comparisons of populations in different categories; e.g., A1–B1, A2–B2, A1–B2, A2–B1). However, this presents a potential pitfall of statistical pseudoreplication due to multiple measurements of the same population. To circumvent the flaw of pseudoreplication, we developed a software package (called “CatPop”) that uses combinatorics to create all possible scenarios of population assignments to categories and perform a permutation test to identify genes where there is greater divergence in “between” comparisons compared to “within” comparisons. We validate CatPop’s statistical approach using a randomly generated dataset of divergence values in 8 populations with 1,000 genes. We then use CatPop to examine genetic divergence in an empirical dataset of 397 genes of interest from six garter snake populations assigned to two categories (a “fast-aging” ecotype and a “slow-aging” ecotype) and identify significant genes in several age-associated gene networks, including pathways of nutrient-sensing, DNA macromolecule damage/repair, and stress adaptation.

	<h2>Physiology</h2>

		<h3>Skeletomuscular Histology</h3>

		<h3>Mitochondrial Respirometry</h3>

	<h2>Evolutionary Ecology</h2>
		
		<h3>Life History Evolution</h3>

</body>

